Assets/Scripts/Model/Cards/Card.cs

```diff
@@ -23,6 +23,7 @@ public abstract class Card
         public bool Faceup { get; private set; } = false;
         public Information Information { get; private set; } = Information.HIDDEN_FROM_ALL;
         public bool Active { get; private set; } = false;
+        public abstract Stealable Stealable { get; }
```

Comment:
Perhaps the best default would be the `Type.Stealable`. This way all non-agenda cards would not be stealable by default. And all agendas are stealable by default. Specific ones, like Obokata Protocol, can still override the `Stealable` property (just like the `virtual TrashOptions`).

Reply:
Ok, I will try to do that :)